.vscode/launch.json

```diff
@@ -22,7 +22,7 @@
             "args": [
                 "-u", "tdd",
 				"--colors",
-				"out/test/**/*.test.js",
+				"out/test/*.test.js",
```

Comment:
I changed this to exclude the integration tests when running the unit tests, which I thought is what this launch config is for. The output was interspersed, and it got confusing because I thought unit tests were failing when really it was the integration tests. I can put it back if you'd like? I just figured since there was a separate launch config to run the integration tests, this might make it more clear what is running

Reply:
Oh yeah that makes sense